includes/payment-methods/class-giropay-payment-method.php

```diff
@@ -24,7 +25,7 @@ class Giropay_Payment_Method extends UPE_Payment_Method {
 	public function __construct( $token_service ) {
 		parent::__construct( $token_service );
 		$this->stripe_id   = self::PAYMENT_METHOD_STRIPE_ID;
-		$this->title       = 'giropay';
+		$this->title       = Payment_Method::GIROPAY;
```

Comment:
This doesn't feel right to me. 

In other `...Payment_Method extends Upe_Payment_Method` classes the title property isn't taken from the corresponding Payment_Method constant. On a quick glance it seems to be because in all other cases the title is capitalised. 

I think of these new constants as `id`s of payment methods and we shouldn't reference an id to a title only because they happen to be exactly the same. 

---

Digging deeper I found some methods have translation strings for title property (For example, `Link_Payment_Method`) making my case stand even more. A title is human readable string representation of payment method and taking it's value from constant that's generally is used as `id` is inconsistent and confusing to a reader.

Reply:
Thanks for your review and time, Karlis! I really appreciate it!  I will check this and have it fixed 🙌

Reply:
Thanks for spotting this. You are right. I have avoided converting titles to constants at all places. I may have missed this one. I will have it changed.

Reply:
Now that you mentioned, I see that there is a [translation tag](https://github.com/Automattic/woocommerce-payments/blob/a19e24ffdc6a2671541d860d532efd584e989e49/includes/payment-methods/class-link-payment-method.php#L28) for the title of `Link` 

Since Link here is actually a brand name of a service of Stripe, will it be better to remove the translation tag from this? In other words `Link` would be read as `Link` itself in any language I guess since it is a brand name. What do you think @KarlisJ ?